src/editor/PointEntRenderer.cpp

```diff
@@ -66,6 +66,7 @@ void PointEntRenderer::genPointEntCubes() {
 
 EntCube* PointEntRenderer::getCubeMatchingProps(EntCube* cube) {
 	for (int i = 0; i < entCubes.size(); i++) {
+		// FIXME : warn? cube filled not fully

```

Comment:
what does this mean? Cubes might not have valid vertex data, but that's ignored here.

Reply:
ok